docs/install-machine.md

```diff
@@ -57,6 +57,21 @@ to your PATH.
         $ docker-machine -v
         machine version 0.5.0 (3e06852)
 
+## Installing bash completion scripts
+
+There are a [number of scripts](../contrib/completion/bash) which add command
```

Comment:
@leedm777 When you run `make docs` to validate your changes --- you are doing this right? :-D --- it reports an error on your link.

```
...
ERROR: 2015/10/22 LinkResolver: No page found for "../contrib/completion/bash" on page "machine/install-machine.md".
...
```
Soon our own @SvenDowideit will add this to the CI and such things will be caught when you submit your PR. For now, you have to run `make docs`

@nathanleclaire I'm checking the added content. Thanks for the ping.